nan_json.h

```diff
@@ -77,15 +77,19 @@ class JSON {
 #if NAN_JSON_H_NEED_PARSE
     return scope.Escape(parse(json_string));
 #else
+    Nan::MaybeLocal<v8::Value> result;
+#if NODE_MODULE_VERSION == 14  // v0.12
```

Comment:
Actually, I think we should change this to `#if NODE_MODULE_VERSION == NODE_0_12_MODULE_VERSION`